functions/go/set-namespace/transformer/namespace.go

```diff
@@ -18,206 +18,311 @@ import (
 	"strings"
 
 	"github.com/GoogleContainerTools/kpt-functions-sdk/go/fn"
+	"k8s.io/apimachinery/pkg/util/sets"
+
```

Comment:
nit: new line is not necessary. If you want to separate github imports from k8s imports, the newline should be after "github.com/GoogleContainerTools/kpt-functions-sdk/go/fn"

Reply:
missed this comments before (GitHub folds comments). 😆  
Done